### PR TITLE
fix competing consumer http api docs

### PR DESCRIPTION
**Pull request by prolic**

### CODE REVIEW

File: docs/http-api/competing-consumers.html [hunk 1]
```diff
@@ -547,15 +547,13 @@ <h2 id="getting-information-for-all-subscriptions">Getting information for all s
...
-<th>Supported Content Types</th>
 <th>Method</th>
 </tr>
 </thead>
 <tbody>
 <tr>
 <td><code>/subscriptions</code></td>
-<td><code>application/json</code></td>
-<td>POST</td>
+<td>GET</td>
```
Comment (prolic) [inline docs/http-api/competing-consumers.html]: this is the real change, the rest of the diff is mostly noise

**Comment by ChrisChinchilla** [comment]
Thanks @prolic we are just about to role out a huge change to the docs which includes a complete switch of documentation system. So when that's complete I will go through PRs snd roll them into that new platform.

**Comment by ChrisChinchilla** [comment]
OK, rolled in @prolic , thanks again!

**Comment by prolic** [comment]
:+1:

**Pull request closed without merging**